src/pages/_document.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import Document, {
+  Html,
+  Head,
+  Main,
+  NextScript,
+  DocumentContext,
+} from "next/document";
+import { DocumentInitialProps } from "next/dist/next-server/lib/utils";
+import { Header } from "../components/Header";
+import { PhaseBanner } from "../components/PhaseBanner";
+import { Footer } from "../components/Footer";
+
+class GovUKTemplate extends Document {
+  static async getInitialProps(
+    ctx: DocumentContext
+  ): Promise<DocumentInitialProps> {
+    const initialProps = await Document.getInitialProps(ctx);
+    return { ...initialProps };
+  }
+
+  render(): React.ReactElement {
+    return (
+      // TODO: https://github.com/madetech/mca-beacons-webapp/issues/25
+      <Html className={"govuk-template "}>
```

Comment:
Is this class name required here? If it's not adding anything I would say remove the class name...

Reply:
Sorry @matthew-a-carr, I should have added why these classes are important.  Without them the layout and background colour of the footer is off (note the whitespace at the bottom):

![image](https://user-images.githubusercontent.com/54271433/106169604-7537c300-6187-11eb-878c-d512de10f34c.png)

Both are from the [GOV.UK Page template](https://design-system.service.gov.uk/styles/page-template/).  

I haven't had the time to look into including the GOV.UK page template to `html` and `body` etc. thoroughly as I wanted to get something set up for the showcase. I raised issue #25 to revisit this (including the console error) tomorrow morning / next week.

Reply:
Okay, happy in that case.  Will just need to raise in planning the issues that we're tracking here as they aren't currently captured on the backlog in Trello and we will need to account at least some time to them for next sprint